TrafficCapture/trafficCaptureProxyServer/README.md

```diff
@@ -54,3 +54,15 @@ Follow documentation for [deploying solution](../../deployment/README.md) to set
     2. Run the following command to list the Kafka topics, and confirm that the `logging-traffic-topic` has been created.
         * For **default/Docker Kafka** clusters `./kafka-tools/kafka/bin/kafka-topics.sh --bootstrap-server "$MIGRATION_KAFKA_BROKER_ENDPOINTS" --list`
         * For **AWS MSK(Kafka)** clusters `./kafka-tools/kafka/bin/kafka-topics.sh --bootstrap-server "$MIGRATION_KAFKA_BROKER_ENDPOINTS" --list --command-config kafka-tools/aws/msk-iam-auth.properties`
+
+## Failure Modes
+
```

Comment:
Can you define what "mutating" vs "non-mutating" are here.

Reply:
Added